modules/ingest-common/build.gradle

```diff
@@ -18,8 +18,8 @@
  */
 
 esplugin {
-    description 'Ingest processor that uses grok patterns to split text'
-    classname 'org.elasticsearch.ingest.grok.IngestGrokPlugin'
+    description 'Module for ingest processors that do not require additional security permissions.'
```

Comment:
> do not require additional security permissions.

is that the actually what *will* be shared by processors that can be found in this module? Isn't the reason we moved geoip into its own plugin because of package size, not because of its security permissions?